Misc/NEWS.d/next/Core and Builtins/2021-12-12-00-15-53.bpo-46053.tnok-6.rst

```diff
@@ -0,0 +1 @@
+Fix OSS audio support on NetBSD
```

Comment:
It should be in the "Library" section, not in "Core and Builtins". And please add a sentence-ending period.

Otherwise LGTM.